src/config.json

```diff
@@ -6,15 +6,19 @@
       "symbol": "BTC",
       "quantumsPerCommon": "100000000",
       "secondsPerBlock": 600,
-      "feeEstimate": "20000"
+      "feeEstimate": "20000",
+      "maxChannelBalance": "16777215",
+      "maxPaymentSize": "4294967"
```

Comment:
Where did you get these values? Can we document somewhere so we know where to find them later

Reply:
@treygriffith could probably just throw a little section in the README under the heading `Config`

Reply:
@treygriffith are you cool with these comments or do you think it makes it messy?